src/server_manager/install_scripts/install_server.sh

```diff
@@ -261,7 +261,7 @@ function start_shadowbox() {
   declare -a docker_shadowbox_flags=(
     --name shadowbox --restart=always --net=host
     -v "${STATE_DIR}:${STATE_DIR}"
-    -e "SB_STATE_DIR=${STATE_DIR}"
+    -e "SB_OUTLINE_DIR=/root/shadowbox"
```

Comment:
This is not backward compatible. The current image needs the SB_STATE_DIR. We need to release a new image before we can make this change. Let's revert this and add a TODO to use SB_OUTLINE_DIR once the image is in production.